### DISCUSSION TITLE
TiTiler at FOSS4G 2022

### DISCUSSION THREAD

**Discussion by vincentsarago** [developmentseed/titiler#505] [Show and tell]
Here are the slides I made for FOSS4G 2022:
https://docs.google.com/presentation/d/1QcvcPEOgFWk_qffkmncdH5GlqN8xzWmhfUY8oxDf-18/edit?usp=sharing

TL&DR:
- Dynamic tiling is great and offers a lot of rendering options
- TiTiler is a suite of python modules which helps creating Raster services
- It's production ready thanks for FastAPI
- It's made so people can customize/extend it
- it's Open Source and we are trying our best to welcome contributors (we are aware of the learning curve because people have to learn about rio-tiler/morecantile...) 
- TiTiler is part of the `Ti*` family alongside `tifeatures` and `timvt`

**Comment by dvd3v** [comment]
Great presentation and awesome font! Thanks for all the work 🙏
